### ISSUE TITLE
mixed-sensor returns "unknown"-state

### ISSUE THREAD

**Issue by Skeletorjus** [custom-components/sensor.avfallsor#9]
The sensor for mixed returns unknown. This is probably because the info is missing on the new site.
AFAIK the mixed sensor should be matching with the bio sensor (they get collected every week), so if there's no other option it could be assumed that mixed is the same value as bio.

**Comment by Hellowlol** [comment]
I can see that the info is missing from your address (your forgot sanitize the code your posted on the last issue, I have removed the address for so it isn't public anymore). 

It would be possible to do what your suggest, but I'd rather wait until the dust settles. I'm don't know what kind of info that will displayed on the website after newyears.

**Comment by Skeletorjus** [comment]
I agree, best to wait and see. I'll just use the bio sensor for now.
Thank you for sanitizing my post.

You can decide whether this should be closed or not.

**Comment by Hellowlol** [comment]
Ill keep it open :)
